docs/setup-guide.md

```diff
@@ -32,6 +32,10 @@ requirements:
  * Each node requires that the `mount.glusterfs` command is available. Under
   all Red Hat-based OSes this command is provided by the `glusterfs-fuse`
   package.
+  
+  * GlusterFS client version installed on nodes should match version of the
+   server which is running inside container. To get installed versions run
```

Comment:
Very close. A more accurate statement would be `should be as close as possible to the version of the server`. If the client is more advanced than the server, for instance, things should still work.